[IMP] fiscal_localizations: Documentation for l10n_it_riba (#11847)
odoo/documentation: content/applications/finance/fiscal_localizations/italy.rst

     government `website <http://www.fatturapa.gov.it/>`_.
   - The :abbr:`CUU (Codice Univoco Ufficio)` must be included in the electronic invoice
     corresponding to the element **1.1.4** (:guilabel:`CodiceDestinario`).

Ri.Ba. (Ricevuta Bancaria)
==========================

> :abbr:`Ri.Ba. (Ricevuta Bancaria)` is a payment method widely used in Italy where businesses request
> payments through their bank, which forwards the request to the debtor's own bank and takes
> responsibility for the collection. This enables payment automation and reduces risks for the vendor.
> 
> The creditor generally uploads a fixed-format text file with the list of payments to the bank's web
> portal.

Reviewer: ```suggestion
:abbr:`Ri.Ba. (Ricevuta Bancaria)` is a payment method widely used in Italy where vendors request
payments through their bank, which forwards the request to the customer's own bank and takes
responsibility for the collection. This enables payment automation and reduces risks for the vendor.

The vendor generally uploads a fixed-format text file with the list of payments to the bank's web
portal.
```
Let's be consistent with using "vendor" and "customer" instead of switching back and forth between those and "creditor"/"debtor".

.. note::
   - Ri.Ba. are exclusively for **domestic payments** in Italy. For recurrent international
     payments, please use `SEPA Direct Debt (SDD) <../accounting/payments/batch_sdd>`_

Configuration
-------------

> - Check that the `l10n_it_riba` module :ref:`is installed <general/install>`.
> 
> - Fill out the required :guilabel:`SIA Code` on the Company settings form.
> 
>   .. image:: italy/sia-code.png
>      :alt: The company's SIA code
> 
>   .. note::
>      The :guilabel:`SIA Code` identifies businesses within the Italian banking network and is used
>      to receive money through specific payment methods. It consists of one letter and four digits
>      (e.g., T1234) and can usually be found on the bank's portal or obtained by contacting the bank.
> 
> - Ensure the Company's bank account has an Italian IBAN.

Reviewer: ```suggestion
#. Check that the `l10n_it_riba` module is :ref:`installed <general/install>`.
#. Go to :menuselection:`Settings --> Users & Companies --> Companies` and select the company that
   will use Ri.Ba.
#. Fill out the required :guilabel:`SIA Code`.

  .. image:: italy/sia-code.png
     :alt: The company's SIA code

  .. note::
     The :guilabel:`SIA Code` identifies businesses within the Italian banking network and is used
     to receive money through specific payment methods. It consists of one letter and four digits
     (e.g., T1234) and can usually be found on the bank's portal or obtained by contacting the bank.

#. Ensure the Company's bank account has an Italian IBAN.
```
Let's use a numbered list since the order of the steps matters, and let's add a step mentioning how to access the company record. No need for blank lines between items in a numbered list (but they are required before/after the image and note).

  .. seealso::
     How to configure :doc:`Bank Accounts <../accounting/bank>`

Accept RiBa for your invoices
-----------------------------

> Payments of type :abbr:`Ri.Ba. (Ricevuta Bancaria)` can be registered in the invoices list
> (:menuselection:`Accounting --> Customers --> Invoices`).

Reviewer: ```suggestion
Payments of type :abbr:`Ri.Ba. (Ricevuta Bancaria)` can be registered from the :guilabel:`Invoices`
list view (:menuselection:`Accounting --> Customers --> Invoices`).
```
